panel/models/deckgl.py

```diff
@@ -49,20 +47,20 @@ def __javascript__(cls):
     def __js_skip__(cls):
         return {
             'deck': cls.__javascript__[:-1],
-            'mapboxgl': cls.__javascript__[-1:]
+            'mapboxgl': cls.__javascript__[-1:],
         }
 
     __js_require__ = {
-        'paths': OrderedDict([
-            ("h3", f"{config.npm_cdn}/h3-js@3.7.2/dist/h3-js.umd"),
-            ("deck-gl", f"{config.npm_cdn}/deck.gl@8.6.7/dist.min"),
-            ("deck-json", f"{config.npm_cdn}/@deck.gl/json@8.6.7/dist.min"),
-            ("loader-csv", f"{config.npm_cdn}/@loaders.gl/csv@3.1.7/dist/dist.min"),
-            ("loader-json", f"{config.npm_cdn}/@loaders.gl/json@3.1.7/dist/dist.min"),
-            ("loader-tiles", f"{config.npm_cdn}/@loaders.gl/3d-tiles@3.1.7/dist/dist.min"),
-            ("mapbox-gl", "https://api.mapbox.com/mapbox-gl-js/v3.0.1/mapbox-gl"),
-            ("carto", "{config.npm_cdn}/@deck.gl/carto@^8.7.0/dist.min.js"),
-        ]),
+        'paths': {
+            "h3": f"{config.npm_cdn}/h3-js@3.7.2/dist/h3-js.umd",
+            "deck-gl": f"{config.npm_cdn}/deck.gl@8.6.7/dist.min",
+            "deck-json": f"{config.npm_cdn}/@deck.gl/json@8.6.7/dist.min",
+            "loader-csv": f"{config.npm_cdn}/@loaders.gl/csv@3.1.7/dist/dist.min",
+            "loader-json": f"{config.npm_cdn}/@loaders.gl/json@3.1.7/dist/dist.min",
+            "loader-tiles": f"{config.npm_cdn}/@loaders.gl/3d-tiles@3.1.7/dist/dist.min",
+            "mapbox-gl": "https://api.mapbox.com/mapbox-gl-js/v3.0.1/mapbox-gl",
+            "carto": "{config.npm_cdn}/@deck.gl/carto@^8.7.0/dist.min.js",
```

Comment:
Probably a bug here with missing f-string?

Don't want to mix it in here, better as separate PR.

Reply:
I'm fine with fixing glaring bugs like this one :+1:

Reply:
Will fix separately.